index.bs

```diff
@@ -8882,34 +8946,34 @@ See the rules for converting ECMAScript values to IDL [=buffer source types=] in
     <pre highlight="webidl">
```

Comment:
Maybe update the intro sentence to explain that you're actually showing all possible combinations.

(Preexisting issue.)